tests/waterservices_test.py

```diff
@@ -280,17 +278,16 @@ def assert_metadata(requests_mock, request_url, md, site, parameter_cd, format):
         site_info, _ = md.site_info()
         assert type(site_info) is DataFrame
     if parameter_cd is None:
-        assert md.variable_info is None
+        assert md.variable_info is None    
     else:
-        pcode_request_url = "https://nwis.waterdata.usgs.gov/nwis/pmcodes/pmcodes?radio_pm_search=pm_search" \
-                            "&pm_group=All%2B--%2Binclude%2Ball%2Bparameter%2Bgroups&pm_search={}" \
-                            "&show=parameter_group_nm%2Ccasrn%2Csrsname%2Cparameter_units%2Cparameter_nm" \
-                            "&format=rdb".format(parameter_cd)
+        pcode_request_url = "https://help.watasdasdasdasdasderdata.usgs.gov/code/parameter_cd_nm_query?fmt=rdb&parm_nm_cd=%25{}%25".format(parameter_cd)
         with open('data/waterdata_pmcodes.txt') as text:
             requests_mock.get(pcode_request_url, text=text.read())
-        variable_info, _ = md.variable_info()
-        assert type(variable_info) is DataFrame
+        #variable_info, _ = md.variable_info()
+        #assert type(variable_info) is DataFrame
```

Comment:
Commenting out these two lines will remove testing for variable_info in all tests.

```
    if 'parameterCd' in parameters:
        md.variable_info = lambda: get_pmcodes(parameterCd=parameters['parameterCd'])
```
https://github.com/UCHIC/dataretrieval/pull/19/files#diff-dd22e9fc6f01be0a9426e13dbfce9c09bd448fd78e866f39e117e563181a57b3R766-R767

Looks like when the parameter_cd paramater is not None, variable_info is set.  Which test did you make this change for?

Reply:
@cjbas22